### PR TITLE
Fixed all compile errors, nulled the iceops update method, added _usl…

### PR DESCRIPTION
**Pull request by MichaelHillcox**
…eep for win32 compile

Pretty simple fixing of the patch and cleaning of some code. Messy old devs throwing issues effectively. The new _usleep should be fine, it's only really used is server based instances like map downloading so lets assume that nobody in their right mind is using the patch on windows... It should be fine. Tested and still works as expected.

### CODE REVIEW

File: src/win32/sys_cod4linker_win32.c [hunk 1]
```diff
@@ -44,6 +44,7 @@
 #include <inttypes.h>
 #include <direct.h>
 #include <windows.h>
+#include "../win32/win32_usleep.h"
```
Comment (atrX) [inline src/win32/sys_cod4linker_win32.c]: You can just import this as `#include "win32_usleep.h"`.

File: src/win32/sys_win32.h [hunk 2]
```diff
@@ -50,4 +50,4 @@ char** PE32_GetStrTable(void *buff, int len, sharedlib_data_t *text);
 
 extern WinVars_t g_wv;
 extern byte cod4_plt[8192];
-#define IDI_ICON1                       1
+#define IDI_ICON1                       1
```
Comment (atrX) [inline src/win32/sys_win32.h:53]: Only change is the removal of a newline, why?

**Review by atrX — CHANGES_REQUESTED** [review]

File: src/win32/sys_win32.h (hunk 2 above)
Comment (MichaelHillcox) [reply src/win32/sys_win32.h:53]: Likely my editor?

File: src/win32/sys_cod4linker_win32.c (hunk 1 above)
Comment (MichaelHillcox) [reply src/win32/sys_cod4linker_win32.c]: Intresting :D do you also not know how to use code suggestions :P 
```suggestion
#include "win32_usleep.h"
```

**Review by atrX — APPROVED** [review]

**Pull request merged**